views/templates/hook/checkout.tpl

```diff
@@ -146,36 +154,36 @@
     </div>
 {/if}
 {if $PS_VERSION === '1-7'}
-<section>
-    <div class="payment-method ps_version_{$PS_VERSION|escape:'htmlall':'UTF-8'}" id="clearpay-method" >
-        <div class="payment-method-content clearpay ps_version_{$PS_VERSION|escape:'htmlall':'UTF-8'}" id="clearpay-method-content">
-            <div class="clearpay-header">
-                <div class="clearpay-header-img">
-                    <img src="{$LOGO_BADGE|escape:'htmlall':'UTF-8'}">
-                </div>
-                <div class="clearpay-header-text1">
-                    {$MORE_HEADER1|escape:'htmlall':'UTF-8'}
+    <section>
+        <div class="payment-method ps_version_{$PS_VERSION|escape:'htmlall':'UTF-8'}" id="clearpay-method" >
+            <div class="payment-method-content clearpay ps_version_{$PS_VERSION|escape:'htmlall':'UTF-8'}" id="clearpay-method-content">
+                <div class="clearpay-header">
+                    <div class="clearpay-header-img">
+                        <img src="{$LOGO_BADGE|escape:'htmlall':'UTF-8'}">
+                    </div>
+                    <div class="clearpay-header-text1">
+                        {$MORE_HEADER1|escape:'htmlall':'UTF-8'}
+                    </div>
+                    <div class="clearpay-header-text2">
+                        {$MORE_HEADER2|escape:'htmlall':'UTF-8'}
+                    </div>
                 </div>
-                <div class="clearpay-header-text2">
-                    {$MORE_HEADER2|escape:'htmlall':'UTF-8'}
+                <div class="clearpay-more-info-text">
+                    <div class="clearpay-more-info">
+                        {$MOREINFO_ONE|escape:'htmlall':'UTF-8'}
+                    </div>
+                    <afterpay-price-table
+                            data-type="price-table"
+                            data-amount="{$TOTAL_AMOUNT|escape:'htmlall':'UTF-8'}"
+                            data-price-table-theme="white"
+                            data-locale="{$ISO_COUNTRY_CODE|escape:'htmlall':'UTF-8'}"
+                            data-currency="{$CURRENCY|escape:'htmlall':'UTF-8'}">
+                    </afterpay-price-table>
```

Comment:
The correct implementation is as below see price [table table](https://js.sandbox.afterpay.com/index.html) tab

```html                    
<afterpay-placement
      data-type="price-table"
      data-amount="{$TOTAL_AMOUNT|escape:'htmlall':'UTF-8'}"
      data-price-table-theme="white"
      data-locale="{$ISO_COUNTRY_CODE|escape:'htmlall':'UTF-8'}"
      data-currency="{$CURRENCY|escape:'htmlall':'UTF-8'}">
</afterpay-placement>
```

Reply:
resolved